src/FSharpSource.targets

```diff
@@ -185,6 +185,7 @@
     <DefineConstants>$(DefineConstants);NETSTANDARD1_6</DefineConstants>
     <DefineConstants>$(DefineConstants);NETCOREAPP1_0</DefineConstants>
     <OtherFlags>$(OtherFlags) --simpleresolution</OtherFlags>
+    <TargetProfile>netcore</TargetProfile>
```

Comment:
@enricosada You should also pass it when compiling up the .NET Portable profiles where System.Runtime is king (everything except the "legacy" profiles like Profile47)